wazo_call_logd/database/alembic/versions/5279fdae3198_add_remote_party_table.py

```diff
@@ -0,0 +1,68 @@
+"""add remote_party_table
+
+Revision ID: 5279fdae3198
+Revises: 4e193f4c53d0
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import HSTORE
+from sqlalchemy.ext.mutable import MutableDict
+
+# revision identifiers, used by Alembic.
+revision = '5279fdae3198'
+down_revision = '4e193f4c53d0'
+
+
+def upgrade():
+    op.create_table(
+        'call_logd_remote_party',
+        sa.Column(
+            'kind',
+            sa.Enum(
+                'user',
+                'conference',
+                'meeting',
+                'unknown',
+                name='call_logd_remote_party_kind',
+            ),
+            nullable=False,
+        ),
+        sa.Column(
+            'description',
+            MutableDict.as_mutable(HSTORE),
```

Comment:
should we indicate `nullable=False` ?
The description is empty {} or contains the id+name.